app/agent/origin.scala

```diff
@@ -26,13 +26,12 @@ class Accounts(prismConfiguration: PrismConfiguration) extends Logging {
   val ArnIamAccountExtractor: Regex = """arn:aws:iam::(\d+):user.*""".r
```

Comment:
`ArnIamAccountExtractor` is unused now, shall we remove?

Reply:
Thank, yes I did, missed that when I split the original PR in two.